misc-scripts/xref_mapping/XrefParser/VGNCParser.pm

```diff
@@ -26,6 +83,13 @@ use Text::CSV;
 
 use parent qw( XrefParser::HGNCParser );
 
+
+=head2 run
+  Description: Runs the VGNCParser
+  Return type: N/A
```

Comment:
Technically, we return an integer here... In practice it's always 0 because refactored parsers are all supposed to abort with an exception unless successful - but we do return something.